src/universal/components/NewMeetingLobby.js

```diff
@@ -112,6 +115,55 @@ const StyledButton = styled(PrimaryButton)({
   width: '100%'
 })
 
+const actionGettingStartedLink = makeExternalLink(
+  'Getting Started Guide',
+  'https://www.parabol.co/getting-started-guide/action-meetings-101'
+)
+const retroGettingStartedLink = makeExternalLink(
+  'Getting Started Guide',
+  'https://www.parabol.co/getting-started-guide/retrospective-meetings-101'
+)
+
+const actionLobbyHelpContent = (
+  <div>
+    <p>
+      {'To learn more about how to run an Action Meeting, see our '}
+      {actionGettingStartedLink}
+      {'.'}
+    </p>
+  </div>
+)
+
+const retroGettingStartedLinkContent = (
+  <p>
+    {'See our '}
+    {retroGettingStartedLink}
+    {' for running a Retrospective Meeting.'}
+  </p>
+)
+
+const retroLobbyHelpContentFree = (
+  <div>
+    <p>{'The person who presses “Start Meeting” will be today’s Facilitator.'}</p>
+    <p>{'Everyone’s display automatically follows the Facilitator.'}</p>
+    {retroGettingStartedLinkContent}
+  </div>
+)
+
+const retroLobbyHelpContentPaid = (
+  <div>
+    <p>
+      {'Running a retrospective is the most effective way to learn how your team can work smarter.'}
```

Comment:
this sounds great, but very sales-driven. is this meant to be for the non-pro folks? i'll go ahead & swap the 2